### PR TITLE
Fix invalid YAML.

### PR DESCRIPTION
**Pull request by dilijev**
FYI @bterlson @akroshg

### CODE REVIEW

File: test/built-ins/TypedArrays/buffer-arg-proto-from-ctor-realm-sab.js [hunk 1]
```diff
@@ -24,8 +24,7 @@ info: |
        b. Let proto be realm's intrinsic object named intrinsicDefaultProto.
     5. Return proto.
 includes: [testTypedArray.js]
-features: [SharedArrayBuffer]
-features: [Reflect]
+features: [SharedArrayBuffer,Reflect]
```
Comment (leobalter) [inline test/built-ins/TypedArrays/buffer-arg-proto-from-ctor-realm-sab.js]: thanks for fixing it!

For consistency, can you just add a space after the comma character?

File: test/built-ins/TypedArrays/buffer-arg-use-custom-proto-if-object-sab.js [hunk 2]
```diff
@@ -31,9 +31,8 @@ info: >
   10. Set the [[Prototype]] internal slot of A to prototype.
   ...
   12. Return A.
-features: [Reflect]
 includes: [testTypedArray.js]
-features: [SharedArrayBuffer]
+features: [SharedArrayBuffer,Reflect]
```
Comment (leobalter) [inline test/built-ins/TypedArrays/buffer-arg-use-custom-proto-if-object-sab.js]: same, thanks!

File: test/built-ins/TypedArrays/buffer-arg-proto-from-ctor-realm-sab.js (hunk 1 above)
Comment (dilijev) [inline test/built-ins/TypedArrays/buffer-arg-proto-from-ctor-realm-sab.js]: Sure. Sorry about that.

**Comment by dilijev** [comment]
@leobalter Fixed formatting.

**Comment by leobalter** [comment]
Thanks!

**Pull request merged**